spec/system/courses/curriculum_spec.rb

```diff
@@ -103,27 +103,27 @@
 
     # All targets should have the right status written next to their titles.
     within("a[aria-label='Select Target #{completed_target_l4.id}']") do
-      expect(page).to have_content('Passed')
+      expect(page).to have_content('Completed')
     end
 
     within("a[aria-label='Select Target #{pending_target_g1.id}']") do
-      expect(page).to have_content('Pending')
+      expect(page).not_to have_content('Pending')
```

Comment:
It would be better to remove this particular test. I don't think we need to check whether we're _not_ doing something... 😅 

There's more instances of this in the file - I think we can get rid of all of 'em.